library_test.py

```diff
@@ -30,8 +32,21 @@ def get_arguments() -> tuple[argparse.ArgumentParser, argparse.Namespace]:
         "--username", "-u", type=str, default="vodafone", help="Set router username"
     )
     parser.add_argument("--password", "-p", type=str, help="Set router password")
+    parser.add_argument(
+        "--configfile",
+        "-f",
```

Comment:
```suggestion
        "-cf",
```

Reply:
I would prefer `-c` then (to keep short versions of params at a single letter), would that be OK?

Reply:
I prefer `cf` like config file. A lot of other params are not 1 single char.

Reply:
ok

Reply:
done